DashSync/shared/Models/DAPI/Networking/DSDAPIPlatformNetworkService.m

```diff
@@ -480,8 +480,8 @@ - (void)loadBloomFilter:(NSString *)filter
     DSPlatformRequestLog(@"fetchIdentitiesByKeyHashes %@", keyHashesArray);
     GetIdentitiesByPublicKeyHashesRequest *getIdentitiesByPublicKeyHashesRequest = [[GetIdentitiesByPublicKeyHashesRequest alloc] init];
     getIdentitiesByPublicKeyHashesRequest.publicKeyHashesArray = [keyHashesArray mutableCopy];
-    getIdentitiesByPublicKeyHashesRequest.prove = DSPROVE_PLATFORM;
-    DSDAPIGRPCResponseHandler *responseHandler = [[DSDAPIGRPCResponseHandler alloc] initForGetIdentitiesByPublicKeyHashesRequest:keyHashesArray withChain:self.chain requireProof:DSPROVE_PLATFORM];
+    getIdentitiesByPublicKeyHashesRequest.prove = NO; 
```

Comment:
I turned it off only for this method (and since it's temporary and we will switch back to YES, I haven't created a special constant, the other methods still use DSPROVE_PLATFORM as YES and it works correctly

Reply:
I have switched DSPROVE_PLATFORM off completely for now and substituted "NO" back with "DSPROVE_PLATFORM"